src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/RestoreTelemetryEvent.cs

```diff
@@ -12,10 +12,12 @@ namespace NuGet.VisualStudio
     /// </summary>
     public class RestoreTelemetryEvent : ActionEventBase
     {
-        public static string RestoreOperationChecks = nameof(RestoreOperationChecks);
-        public static string PackagesConfigRestore = nameof(PackagesConfigRestore);
-        public static string SolutionDependencyGraphSpecCreation = nameof(SolutionDependencyGraphSpecCreation);
-        public static string PackageReferenceRestoreDuration = nameof(PackageReferenceRestoreDuration);
+        public const string RestoreOperationChecks = nameof(RestoreOperationChecks);
+        public const string PackagesConfigRestore = nameof(PackagesConfigRestore);
+        public const string SolutionDependencyGraphSpecCreation = nameof(SolutionDependencyGraphSpecCreation);
```

Comment:
Why are all this strings public?

Reply:
Do'h! They are used from another assembly :)

Reply:
yep :D 
I'm using the same name the etw events.